### ISSUE TITLE
gui/design Feature Request: Option to change default settings

### ISSUE THREAD

**Issue by DeXoteric** [DFHack/dfhack#3216]
It would be very nice to be able to change default values for options like auto-commit, priority, thickness, spacing

**Comment by myk002** [comment]
@johncosker 

this could be achieved with a json settings file: `dfhack-config/design.json`. You can save simple settings in there. See how `gui/launcher` saves its screen position for an example.

If we decide that any of these settings should be tied to the *fort* instead of global for the user, you can instead use the persistence API. See `prioritize.lua` for an example of that. I don't think fort-specific persistence will be required here, but just making you aware of the options in case it comes up.

**Comment by johncosker** [comment]
@myk002 yeah I've been thinking about this since the first versions of the tool, thanks for the tips. I'd imagine global settings for the user would be fine.

I'm super busy right now with moving so I haven't been able to make updates, but I'll get back on it once things are settled.
